core/templates/domain/collection/read-only-collection-backend-api.service.ts

```diff
@@ -112,7 +112,7 @@ export class ReadOnlyCollectionBackendApiService {
    * rejection callback function is passed the error that occurred and the
    * collection ID.
    */
-  fetchCollection(collectionId: string): Promise<Collection> {
+  async fetchCollectionAsync(collectionId: string): Promise<Collection> {
```

Comment:
Should we make this change in all the places this method is used in the codebase?

Reply:
Hi @Nik-09, is this method not used anywhere else? I only see changes in the spec files ...

Please double-check.

Reply:
Yes, this method is not used anywhere else. PTAL
Thanks.

Reply:
Oh great! Thanks for checking. In that case, can we get rid of this method and the relevant test (since it is dead code anyway)?

Reply:
Done .
PTAL